### PR TITLE
Add bugreport page

### PR DESCRIPTION
**Pull request by Tzky**
This PR adds a but report page to guide new users. 
This will probably get referenced by the contrib docs PR.

### CODE REVIEW

File: source/server/spongineer/bugreport.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
+Reporting Bugs
+==============
+
+If you've encountered a bug and you're unsure on how to report it, this is the page to look at.
+We're currently handling all bug reports through our Issuetracker on Github and via the forums.
```
Comment (kashike) [inline source/server/spongineer/bugreport.rst]: "through our Issuetracker on Github" -> "through our issue trackers on GitHub"

File: source/server/spongineer/bugreport.rst [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+If there's an issue with SpongeForge, report it on the `SpongeForge issue tracker <https://github.com/spongepowered/SpongeForge/issues>`_.
+The same applies for SpongeVanilla (`SpongeVanilla issue tracker <https://github.com/spongepowered/SpongeVanilla/issues>`_)
+and even the Docs (`SpongeDocs issue tracker <https://github.com/spongepowered/SpongeDocs/issues>`_).
+
+Whenever you report a bug, please include the following:
+
+* Sponge version used (SpongeForge or SpongeVanilla build number)
+* Forge version
```
Comment (kashike) [inline source/server/spongineer/bugreport.rst]: (if applicable)

File: source/server/spongineer/bugreport.rst [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+* any other mods installed
```
Comment (kashike) [inline source/server/spongineer/bugreport.rst]: plugins/mods

File: source/server/spongineer/bugreport.rst [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+* the Sponge plugins you're using
```
Comment (kashike) [inline source/server/spongineer/bugreport.rst]: see above, can merge there two items?

File: source/server/spongineer/bugreport.rst [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+* your logfiles
```
Comment (kashike) [inline source/server/spongineer/bugreport.rst]: "log files"?

**Comment by Spongy** [comment]
Setting up PR reference, please wait...

File: source/server/spongineer/bugreport.rst [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+problem still persists, then file a bug report.
+
+Here's a short example, you can copy and use it when opening a ticket on GitHub:
+
+.. code-block:: none
+
+ Build number: #buildnumber here#
+ Forge version: #Forgeversion here#
+
+ Logfile: #link to pastebin here#
```
Comment (kashike) [inline source/server/spongineer/bugreport.rst]: Multiple logs can be provided - "Log files"

File: source/server/spongineer/bugreport.rst [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+* any other mods or plugins installed including their versions
+* your :doc:`logs`
+
+ * please use a service like `GitHubs gist <http://gist.github.com/>`_ or `pastebin <http://pastebin.com/>`_ to provide
+   logs, *don't* paste them into the post directly
+
+Please make sure you already read through our :doc:`debugging`, :doc:`troubleshooting` and :doc:`logs` sections. If the
+problem still persists, then file a bug report.
+
+Here's a short example, you can copy and use it when opening a ticket on GitHub:
```
Comment (kashike) [inline source/server/spongineer/bugreport.rst]: Mention that using this is optional, and that they are free to change it?

File: source/server/spongineer/bugreport.rst [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+.. code-block:: none
+
+ Build number: #buildnumber here#
+ Forge version: #Forgeversion here#
+
+ Logfile: #link to pastebin here#
+ Crashreport: #link to pastebin here#
```
Comment (kashike) [inline source/server/spongineer/bugreport.rst]: A crash report is optional, should mention this?

File: source/server/spongineer/bugreport.rst [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+
+Here's a short example, you may copy and use it when opening a ticket on GitHub. Note that this is just a suggestion,
+its usage is optional and you may modify it to suit your needs:
+
+.. code-block:: none
+
+ Build number: #buildnumber here#
+ Forge version: #Forgeversion here#
+
+ Logfiles: #link to pastebin here#
```
Comment (kashike) [inline source/server/spongineer/bugreport.rst]: "Log files"

File: source/server/spongineer/bugreport.rst [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+ Crashreport: #link to pastebin here#
```
Comment (kashike) [inline source/server/spongineer/bugreport.rst]: "Crash report"

**Pull request merged**